util/db/repository.go

```diff
@@ -3,29 +3,23 @@ package db
 import (
 	"fmt"
 	"hash/fnv"
-	"strings"
+
+	log "github.com/sirupsen/logrus"
 
 	"golang.org/x/net/context"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
-	apiv1 "k8s.io/api/core/v1"
-	apierr "k8s.io/apimachinery/pkg/api/errors"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-
-	log "github.com/sirupsen/logrus"
 
-	"github.com/argoproj/argo-cd/v2/common"
 	appsv1 "github.com/argoproj/argo-cd/v2/pkg/apis/application/v1alpha1"
-	"github.com/argoproj/argo-cd/v2/util/errors"
-	"github.com/argoproj/argo-cd/v2/util/git"
-	"github.com/argoproj/argo-cd/v2/util/settings"
 )
 
 const (
 	// Prefix to use for naming repository secrets
 	repoSecretPrefix = "repo"
 	// Prefix to use for naming credential template secrets
 	credSecretPrefix = "creds"
+	// Prefix to use for naming credential template configuration secrets
+	credConfigSecretPrefix = "repocreds"
```

Comment:
Can we keep using old `credSecretPrefix` please?

Reply:
I noticed that as well just now! Good catch!